tasks/install.yml

```diff
@@ -39,6 +39,7 @@
     name: "git+{{ monasca_agent_git_repo }}@{{ monasca_agent_branch }}"
     virtualenv: "{{ monasca_virtualenv_dir }}"
     editable: False
+    extra_args: "{{ monasca_pip_extra_args }}"
```

Comment:
I don't see this variable listed in defaults, or mentioned in the README. At the moment I'd expect it to fail if you don't specify the argument. Would you use it for passing upper-constraints? Should it also be passed to the preceding to tasks?

Reply:
upper-constraints is the main one. I've addressed comments.